assignment/final/aLife/code/master.css

```diff
@@ -106,7 +108,10 @@ img {
 
 #leftMiddle {
     margin-left: 45px;
-    margin-top: 15px;
+<<<<<<< HEAD
+    top: 15px;
+=======
+>>>>>>> e2a461c30e5dcbeafdc5e29cbb11aa48b3370681
```

Comment:
So this is a *merge conflict*  that `git` didn't auto-resolve.  The `=======` are separating the top (current) changes vs the bottom (incoming) changes.  It wants you to either pick the top, pick the bottom...or manually merge them together. 

Usually vscode will show a special editor that will lets you click a button to resolve...you will probably just need to manually clean this up now.